security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisResultBuilder.java

```diff
@@ -239,6 +239,14 @@ public class PostContingencyResultBuilder extends AbstractLimitViolationsResultB
 
         private PostContingencyComputationStatus status;
 
+        private int createdComponentCount;
+
+        private double lossOfLoad;
```

Comment:
In my opinion, you must add somewhere "activePower" for load and for generation.

Reply:
agree

i would suggest not having total `lossOfLoad` nor  `lossOfGeneration`, only the details of the disconnected equipments, from which and API users can derive their own summaries (total counts, total counts by type, total activePower, totalActivePower by type). Something like:

```
"outagedInjections" : [
  {
    "type": "LOAD",
    "id": "Load1",
    "activePower": 12.34,
    "reactivePower": 2.34
  },
  {
    "type": "GEN",
    "id": "Gen1",
    "activePower": 12.34,
    "reactivePower": 2.34
  },
  {
    "type": "BATTERY",
    "id": "Batt1",
    "activePower": 12.34,
    "reactivePower": 2.34
  }
]
```

Reply:
Yes, it could be nice of course. But keep in mind that it can decrease the performances and increase the complexity in the implementation:
- For loads, in case of slack distribution on loads, what are the values for `P0` and `Q0` ? The base case before slack distribution of after slack distribution ? Furthermore, we also have a mode that keeps the power factor constant, changing reactive power during slack distribution.
- For generators, in case of slack distribution on generation, what are the values of active power ? The values before or after slack distribution ? In case of a generator that controls voltage: we also have to update the reactive power during the security analysis. I imagine we have to put the reactive power at the end of the base case resolution. Right? 
So I think that we must, if we do that, ask for the state after the first load flow, all the regulations done. If yes, I prefer that we try to implement it before merging these PR.

Reply:
I am ok to leave as it is now for this PR. Just some thought for future work 
- for postcontingency results: active power values reported should be the 'cut values' at the time the contingency occurred. so these are basecase state solved value (after slack distribution)
- for post operator strategy results: active power values reported should be the 'cut values' at the time the operator strategy are applied. So these are contingency state solved value (after slack distribution)
- we could probably forget the reactivePower part which doesn't bring much value

Reply:
Good idea for the reactive power part ;-) @geofjamg what do you prefer for active power losses? The detailed solution or the global one?

Reply:
We can start we global/sum values and see later if one really need detailed results

Reply:
I think we have a naming problem: when first looking at loadActivePowerLoss I understand "loss" as heat loss not loss because of lost elements.